src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -37,8 +37,11 @@ public class GitTfsRemote : IGitTfsRemote
             Id = info.Id;
             TfsUrl = info.Url;
             TfsRepositoryPath = info.Repository;
-            TfsUsername = info.Username;
-            TfsPassword = info.Password;
+            // if the command line parameters are passed in, use those. else default to the values in the cofig
```

Comment:
Spelling error.
```suggestion
            // If the command line parameters are passed in, use those. Otherwise use the values from the config.
```